apps/mosaico/src/Article/Article.purs

```diff
@@ -125,6 +134,41 @@ render { props, state, setState } =
         ]
       }
   where
+    renderAuthors authors =
+      DOM.div
+        { className: "mosaico--article-authors"
+        , children:
+            map (DOM.span_ <<< Array.singleton <<< DOM.text <<< _.byline) authors
+            `snoc` premiumBadge
+        }
+      where
+        premiumBadge =
+          guard (maybe false (_.premium <<< fromFullArticle) state.article)
+          DOM.div
+            { className: "mosaico--article--premium background-hbl"
+            , children: [ DOM.text "premium" ]
+            }
+
+    articleTimestamps maybeUpdateTime (LocalDateTime pubTime) =
+      DOM.div
+        { className: "mosaico--article-timestamps"
+        , children:
+            [ publishingTime pubTime
+            , foldMap updateTime maybeUpdateTime
+            ]
+        }
+      where
+        publishingTime time =
+          DOM.div
+            { className: "mosaico--article-published-timestamp"
+            , children: [ DOM.text $ "Pub. " <> formatArticleTime time ]
```

Comment:
Is `"Pub."` Swedish?

Reply:
Yep. This follows roughly the same layout as the apps:
https://app-article.ksfmedia.fi/?uuid=356cb203-0d2d-486e-b66d-e59f5e1021bc&paper=hbl&mode=normal